src/scripts/main.js

```diff
@@ -1,7 +1,43 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const block = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  block.classList.add('notification');
```

Comment:
You can add not only one class to an element. 
So, just ad a type as a second class here